api/service.py

```
@@ -243,6 +241,13 @@ async def lifespan(app: FastAPI):
         except Exception as e:
             print(f"Warning: Observability initialization failed: {e}")
 
+    # Initialize audit logging
+    try:
+        audit_logger = get_audit_logger()
```

Ruff (0.14.10)

Local variable `audit_logger` is assigned to but never used

Remove assignment to unused variable `audit_logger`

(F841)